src/sentry/search/events/filter.py

```diff
@@ -967,14 +975,42 @@ def _project_slug_filter_converter(
 
     def format_search_filter(self, term: SearchFilter) -> Optional[WhereType]:
         """For now this function seems a bit redundant inside QueryFilter but
-        most of the logic from hte existing format_search_filter hasn't been
+        most of the logic from the existing format_search_filter hasn't been
         converted over yet
         """
-        converted_filter = self.convert_search_filter_to_condition(term)
-        if converted_filter:
-            return converted_filter
+        name = term.key.name
+        value = term.value.value
+
+        if name == ISSUE_ALIAS:
+            operator = term.operator
+            value = to_list(value)
+            # `unknown` is a special value for when there is no issue associated with the event
+            group_short_ids = [v for v in value if v and v != "unknown"]
+            filter_values = ["" for v in value if not v or v == "unknown"]
+
+            if group_short_ids and self.params and "organization_id" in self.params:
+                try:
+                    groups = Group.objects.by_qualified_short_id_bulk(
+                        self.params["organization_id"],
+                        group_short_ids,
+                    )
```

Comment:
We can, though I suspect the gains here would be rather tiny since it's unlikely that a single query is going to filter for the same group id more than once.

If we're able to query them in bulk, we might see a bigger improvement though given how the search is transformed, that may not be worth the effort.

Reply:
Agreed, definitely a non-blocker for this change, and a bulk query will have a larger impact :+1:

Reply:
Leaving this as is, if we decide it's worth revisiting, we will 👍